packages/box/box.1.2.0/opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Render boxes in the terminal"
+description: """
+Render boxes in the terminal, can configure the padding, margin and the border of the box.
+"""
+maintainer: ["davesnx"]
```

Comment:
Any chance you could add some sort of email address for people who don't use github?

Reply:
```suggestion
maintainer: ["davesnx <dsnxmoreno@gmail.com>"]
```